packages/runtimes/js/src/JSRuntime.js

```diff
@@ -475,6 +475,24 @@ function getLoaderRuntime({
     loaderCode = `(${loaderCode})`;
   }
 
+  if (needsEsmLoadPrelude && options.featureFlags.importRetry) {
+    const assetId = bundleGraph.getAssetById(bundleGroup.entryAssetId);
+    loaderCode = `
+      Object.defineProperty(module, 'exports', { get: () => {
+        let load = require('./helpers/browser/esm-js-loader-retry');
```

Comment:
How come this isn't using the logic from below? Specifically the non-scope hoisting behaviour of `module.bundle.root`